Bug: rich now imports Console from console module instead of logging

### Expected behavior
The following error was encountered with a CI run:
```python
Traceback (most recent call last):
  File "/opt/hostedtoolcache/Python/3.11.15/x64/bin/scraper", line 3, in <module>
    from pdc_osti.scraper import main
  File "/home/runner/work/pdc-osti/pdc-osti/src/pdc_osti/scraper.py", line 14, in <module>
    from .logger import pdc_log, script_log_end, script_log_init
  File "/home/runner/work/pdc-osti/pdc-osti/src/pdc_osti/logger.py", line 8, in <module>
    from rich.logging import Console, RichHandler
ImportError: cannot import name 'Console' from 'rich.logging' (/opt/hostedtoolcache/Python/3.11.15/x64/lib/python3.11/site-packages/rich/logging.py)
Error: Process completed with exit code 1.
```

This occurred with:
https://github.com/Textualize/rich/commit/b17c6169a4188d4a0ecb0153072a0061e62b8ec7

--- a/.github/workflows/linting.yml
+++ b/.github/workflows/linting.yml
@@ -8,10 +8,10 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - name: Set up Python 3.10
+    - name: Set up Python 3.13
       uses: actions/setup-python@v1
       with:
-        python-version: '3.10'
+        python-version: '3.13'
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip

--- a/.github/workflows/smoke-test.yaml
+++ b/.github/workflows/smoke-test.yaml
@@ -11,7 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: ['3.10', '3.11']
+        python-version: ['3.13', '3.14']
 
     steps:
     - uses: actions/checkout@v2

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -17,7 +17,7 @@ repos:
         # supported by your project here, or alternatively use
         # pre-commit's default_language_version, see
         # https://pre-commit.com/#top_level-default_language_version
-        language_version: python3.10
+        language_version: python3.13
   - repo: https://github.com/pycqa/flake8
     rev: 5.0.4
     hooks:

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -25,7 +25,7 @@ repos:
         args:
           - "--config=setup.cfg"
   - repo: https://github.com/pycqa/isort
-    rev: 5.10.1
+    rev: 8.0.1
     hooks:
       - id: isort
         name: isort (python)

--- a/pyproject.toml
+++ b/pyproject.toml
@@ -14,11 +14,12 @@ maintainers = [
 ]
 classifiers = [
     "Development Status :: 4 - Beta",
-    "Programming Language :: Python :: 3.8",
-    "Programming Language :: Python :: 3.9",
-    "Programming Language :: Python :: 3.10"
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13"
 ]
-requires-python = ">=3.8"
+requires-python = ">=3.10"
 
 dependencies = [
     "requests",
@@ -52,4 +53,4 @@ version = {attr = "pdc_osti.__version__"}
 
 [tool.black]
 line-length = 88
-target-version = ["py310"]
+target-version = ["py313"]

--- a/src/pdc_osti/__init__.py
+++ b/src/pdc_osti/__init__.py
@@ -3,4 +3,4 @@ __version__ = "0.1.0"
 LOG_NAME = "pdc-osti"
 
 PDC_QUERY = {"per_page": 20}
-PDC_URI = "https://pdc-discovery-prod.princeton.edu/discovery/pppl_reporting_feed.json"
+PDC_URI = "https://datacommons.princeton.edu/discovery/pppl_reporting_feed.json"

--- a/src/pdc_osti/logger.py
+++ b/src/pdc_osti/logger.py
@@ -5,8 +5,8 @@ from logging.config import dictConfig
 from pathlib import Path
 
 import yaml
-from rich.logging import Console, RichHandler
-
+from rich.console import Console
+from rich.logging import RichHandler
 from . import LOG_NAME
 from . import __version__ as pdc_osti_version
 

